jsonnet/main.jsonnet

```diff
@@ -79,7 +79,7 @@ local suffix = {
 // create a path including the file name based on the item.
 local path(item) =
   // CRDs go into cert-manager-crds directory
-  if item.kind == 'CustomResourceDefinition' then 'cert-manager-crds/' + item.metadata.name + '-' + suffix[item.kind] + '.yaml'
+  if item.kind == 'CustomResourceDefinition' then '../bundle/manifests/' + item.metadata.name + '-' + suffix[item.kind] + '.yaml'
```

Comment:
is it hard to switch to the root directory instead of it being `bindata`. we could then write `bundle/manifests/` for the crd generation target and `bindata/cert-manager-deployment/` and `bindata/cert-manager-deployment/` for the deployment manifests.

Reply:
it would make it more explicit here.

Reply:
Sure, done.